resources/views/tag.blade.php

```diff
@@ -0,0 +1,88 @@
+
+                                <a class="avatar" href="#">
+                                    <img src="{{ asset('bower/adminbsb-materialdesign/images/image-gallery/15.jpg') }}"
+                                        alt="Profile Image">
+                                </a>
+                                <div class="blog-info">
+                                    <h4 class="title"><a
+                                            href="{{ route('post', $post->id) }}"><b>{{ $post->title }}</b></a><br><br>
+                                        <a class="active" href="{{ route('post', $post->id) }}">{{ trans('Read more') }}
+                                            &raquo;</a>
+                                    </h4>
+                                    @guest
+                                        <ul class="post-footer">
+                                            <li>
+                                                <a href="{{ route('login') }}"><i class="ion-heart"></i>
+                                                    {{ $post->favorite_to_users()->count() }}
+                                                </a>
+                                            </li>
+                                            <li>
+                                                <a href="{{ route('login') }}"><i
+                                                        class="ion-chatbubble"></i>{{ $post->comments()->count() }}</a>
+                                            </li>
+                                            <li>
+                                                <a href="{{ route('login') }}"><i
+                                                        class="ion-eye"></i>{{ $post->view_count }}</a>
+                                            </li>
+                                        </ul>
+                                    @else
+                                        <ul class="post-footer">
+                                            <li>
+                                                <a href="{{ route('post', $post->id) }}"><i class="ion-heart"></i>
+                                                    {{ $post->favorite_to_users()->count() }}
+                                                </a>
+                                            </li>
+                                            <li>
+                                                <a href="{{ route('post', $post->id) }}"><i
+                                                        class="ion-chatbubble"></i>{{ $post->comments()->count() }}</a>
+                                            </li>
+                                            <li>
+                                                <a href="{{ route('post', $post->id) }}"><i
+                                                        class="ion-eye"></i>{{ $post->view_count }}</a>
+                                            </li>
```

Comment:
indent 4 spaces

Reply:
chưa sửa indent nè anh ơi 
```suggestion
                                                <a href="{{ route('post', $post->id) }}"><i
                                                    class="ion-chatbubble"></i>{{ $post->comments()->count() }}</a>
                                            </li>
                                            <li>
                                                <a href="{{ route('post', $post->id) }}"><i
                                                     class="ion-eye"></i>{{ $post->view_count }}</a>
                                            </li>
```